src/models/converters/transactions/tests/summary.rs

```diff
@@ -1,419 +1,205 @@
-            safe_tx_hash: String::from("0xf94fc3d3607845b3e04e2c6ae51802ffe2b4d39dd9b3159a8d9d9e31a05b5beb"),
-            executor: Some(String::from("0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0")),
-            is_executed: true,
-            is_successful: Some(true),
-            eth_gas_price: Some(String::from("1000000000")),
-            gas_used: Some(69431),
-            fee: Some(String::from("69431000000000")),
-            origin: None,
-            confirmations_required: Some(3),
-            confirmations: Some(vec!(
-                Confirmation {
-                    owner: String::from("0x65F8236309e5A99Ff0d129d04E486EBCE20DC7B0"),
-                    submission_date: Some(expected_date),
-                    transaction_hash: None, 
-                    signature_type: String::from("EOA"),
-                    signature: Some(String::from("0x00000000000000000000000065f8236309e5a99ff0d129d04e486ebce20dc7b0000000000000000000000000000000000000000000000000000000000000000001")),
-                },
-                Confirmation {
-                    owner: String::from("0xBEA2F9227230976d2813a2f8b922c22bE1DE1B23"),
-                    submission_date: Some(expected_date),
-                    transaction_hash: None, 
-                    signature_type: String::from("EOA"),
-                    signature: Some(String::from("0xccbf629ad44c41c39e397d4c3c199593be908edf57d728204c2ebd5f384ad5c90f1e7fca86dab291e406724b99d7d956acaa01cb78d146e410a7accd1ffe0df01b")),
-                },
-                Confirmation {
-                    owner: String::from("0xF2CeA96575d6b10f51d9aF3b10e3e4E5738aa6bd"),
-                    submission_date: Some(expected_date),
-                    transaction_hash: None, 
-                    signature_type: String::from("EOA"),
-                    signature: Some(String::from("0x9b3733a7018e90ca4ebb1504e1c230260f8d41d0542180e915fa10af694f18b360f5fe6c9dc252e257b3d45071b32977224122165ff771a018a392bb62dbf1491b")),
-                },
-            )),
-            signatures: Some(String::from("0x00000000000000000000000065f8236309e5a99ff0d129d04e486ebce20dc7b0000000000000000000000000000000000000000000000000000000000000000001ccbf629ad44c41c39e397d4c3c199593be908edf57d728204c2ebd5f384ad5c90f1e7fca86dab291e406724b99d7d956acaa01cb78d146e410a7accd1ffe0df01b9b3733a7018e90ca4ebb1504e1c230260f8d41d0542180e915fa10af694f18b360f5fe6c9dc252e257b3d45071b32977224122165ff771a018a392bb62dbf1491b")),
-        };
-        let token_info = Some(TokenInfo {
-            token_type: TokenType::Erc20,
-            address: String::from("0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85"),
-            name: String::from("Maker"),
-            symbol: String::from("MKR"),
-            decimals: 18,
-            logo_uri: Some(String::from("https://gnosis-safe-token-logos.s3.amazonaws.com/0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85.png")),
+use crate::models::converters::transactions::{data_size};
```

Comment:
I think this is where you need to have `#[cfg(test)]` . Could be the reason Travis is complaining.

Reply:
nope, I was just missing 2 files that's why it is complaining ;)